leveldb/leveldbSerial.go

```diff
@@ -262,6 +297,10 @@ func (s *SerialDB) putBatch() error {
 	result := <-ch
 	close(ch)
 
+	s.mutBatch.Lock()
```

Comment:
not a great solution, this can just set to nil the writing batch if another go-routine called putBatch and managed to set the writingBatch

Reply:
right, reverted